### PR TITLE
Fetch latest version from maven-central

### PR DESCRIPTION
**Pull request by MasseGuillaume**

### CODE REVIEW

File: website/content/docs/installation.md [hunk 1]
```diff
@@ -38,13 +38,9 @@ Homebrew also automatically installs completions for zsh and Bash.
...
 
-```sh
-$ curl -L https://github.com/scalacenter/bloop/releases/download/v1.0.0-M5/install.py | python2
-```
-
-<span class="label warning">Remember</span> to replace the version in the URL
-by the latest version, which is [![Latest
-version](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop/latest.svg)](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop).
+<pre><code class="language-sh">
+$ curl -L https://github.com/scalacenter/bloop/releases/download/<i class="latest-version">1.0.0-M7</i>/install.py | python2
```
Comment (Duhemm) [inline website/content/docs/installation.md]: I meant here

File: website/content/docs/installation.md [hunk 2]
```diff
@@ -77,13 +73,11 @@ the build).
...
 
-```scala
-addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "<latest-release>")
-```
 
-<span class="label warning">Remember</span> to replace `<latest-release>` by
-[![Latest
-version](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop/latest.svg)](https://index.scala-lang.org/scalacenter/bloop/sbt-bloop).
+<pre><code class="language-scala">
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "<i class="latest-version">1.0.0-M7</i>")
```
Comment (Duhemm) [inline website/content/docs/installation.md]: And here please

File: website/content/docs/installation.md (hunk 1 above)
Comment (olafurpg) [reply website/content/docs/installation.md]: might wanna use a different default than `1.0.0-M7` in case people disable js for example

Comment (jvican) [reply website/content/docs/installation.md]: We'll deal with that when we have tut. For now, we'll manually replace that for every release.

**Pull request merged**